samples/net/throughput_server/prj_frdm_k64f.conf

```diff
@@ -16,7 +16,7 @@ CONFIG_PRINTK=y
 
 CONFIG_NET_PKT_RX_COUNT=500
 CONFIG_NET_PKT_TX_COUNT=5
-CONFIG_NET_BUF_RX_COUNT=720
+CONFIG_NET_BUF_RX_COUNT=710
```

Comment:
Is this change really *required* in this commit? I'd say it only bloats up it, this value could easily stay as is. (Mostly rhetoric note, not worth changing, but IMHO not worth to have been included in the first place.)

Reply:
It is required to get that test pass for frdm_k64f. net_pkt has grown a bit, thus this issue.
It's growing even more in next PR #11775, but it's all temporary: in the end it will shrink.

Reply:
> It is required to get that test pass for frdm_k64f

ack, thanks for explaining

Reply:
Shouldn't we change the CONFIG_NET_PKT_RX_COUNT instead here as that is tied to net_pkt? Probably does not matter much actually.